cadquery/occ_impl/geom.py

```diff
@@ -852,3 +858,72 @@ def isInside(self, b2: "BoundBox") -> bool:
             return True
         else:
             return False
+
+
+class Location(object):
+    """Relative location in 3D space"""
```

Comment:
Should there be more background info in this docstring?

Reply:
Yep, I added more info.